README.md

```diff
@@ -1,4 +1,97 @@
-# FLOCX Marketplace
-Just a simple Flask application for FLOCX Marketplace project.
-[More details on FLOCX](https://github.com/CCI-MOC/flocx)
+# flocx-market
```

Comment:
Let's keep the title the same - 'FLOCX Marketplace'

Reply:
Okay.